graylog2-web-interface/src/views/components/WidgetQueryControls.test.jsx

```diff
@@ -58,11 +56,19 @@ describe('WidgetQueryControls', () => {
   const emptyGlobalOverride = {};
   const globalOverrideWithQuery = { query: { type: 'elasticsearch', query_string: 'source:foo' } };
 
-  const renderSUT = (props = {}) => render(
+  const Wrapper = ({ children }: { children: React.Node }) => (
     <WrappingContainer>
+      <SearchBarForm initialValues={{ timerange: { type: 'relative', range: 300 }, queryString: '', streams: [] }} onSubmit={() => {}}>
```

Comment:
You can use `DEFAULT_TIMERANGE` from `views/Contsants` here.

Reply:
:white_check_mark: